internal/setup/setup.go

```diff
@@ -534,14 +535,34 @@ func parsePackage(baseDir, pkgName, pkgPath string, data []byte) (*Package, erro
 			Scripts: SliceScripts{
 				Mutate: yamlSlice.Mutate,
 			},
+			Essential: map[SliceKey]bool{},
+		}
+		for _, refName := range yamlPkg.Essential {
+			sliceKey, err := ParseSliceKey(refName)
+			if err != nil {
+				return nil, fmt.Errorf("invalid slice reference %q in %s", refName, pkgPath)
```

Comment:
"package %q has invalid essential slice reference: %q"

This makes it clear what is what.